deployment/base/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: languagedepot
+
+resources:
+- config-map.yaml
+- secrets.yaml
```

Comment:
I'm not sure what the fall out is, but if I comment this out then I can run skaffold without modifying the base/secrets file.

Reply:
I ended up changing how we do secrets for dev since most of them aren't secret.